src/core/atomic.d

```diff
@@ -893,292 +892,289 @@ version (unittest)
         testLoadStore!(MemoryOrder.raw, T)(val);
     }
 
-    @betterC @safe pure nothrow unittest
-    {
```

Comment:
Why not keep these blocks to signify that they different tests?
(Bonus: That would imply a much smaller diff and does not screw with the line history)